### ISSUE TITLE
Bug on new install

### ISSUE THREAD

**Issue by trollboy** [RobBrazier/svelte-awesome#995]
Code:
```
    import Icon from 'svelte-awesome';
    import calendarCheckO from 'svelte-awesome/icons/calendarCheckO';
    ...
    <Icon data="{calendarCheckO}" />
```

Result
```
Unknown file extension ".svelte" for ~/frontend/node_modules/svelte-awesome/components/Icon.svelte

TypeError [ERR_UNKNOWN_FILE_EXTENSION]: Unknown file extension ".svelte" for ~/frontend/node_modules/svelte-awesome/components/Icon.svelte
    at new NodeError (node:internal/errors:399:5)
    at Object.getFileProtocolModuleFormat [as file:] (node:internal/modules/esm/get_format:79:11)
    at defaultGetFormat (node:internal/modules/esm/get_format:121:38)
    at defaultLoad (node:internal/modules/esm/load:81:20)
    at nextLoad (node:internal/modules/esm/loader:163:28)
    at ESMLoader.load (node:internal/modules/esm/loader:605:26)
    at ESMLoader.moduleProvider (node:internal/modules/esm/loader:457:22)
    at new ModuleJob (node:internal/modules/esm/module_job:64:26)
    at #createModuleJob (node:internal/modules/esm/loader:480:17)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:434:34
```

**Comment by eugbyte** [comment]
Same, am facing the same error

**Comment by eugbyte** [comment]
Reverting to version [3.2.0](https://github.com/RobBrazier/svelte-awesome/releases/tag/v3.2.0) works for now.

```
"dependencies": {
  "svelte-awesome": "3.2.0"
}
```

**Comment by RobBrazier** [comment]
I've updated the dependencies to the latest svelte and @sveltejs/package versions now and released as 3.3.0 - 
I cannot replicate this issue, works fine with a brand new sveltekit project
Please create a new issue with more information if you still face the same problem on the latest version
